apps/io-func-sign-user/openapi.yaml

```
@@ -745,6 +745,8 @@ components:
             - description
         dossier_id:
           $ref: "#/components/schemas/Id"
+        dossier_title:
+          type: string
```

if it is a `NonEmptyString`!
```suggestion
          type: string
          minLength: 1
```